config.sh

```diff
@@ -6,7 +6,7 @@
 function pre_build {
     # Any stuff that you need to do before you start building the wheels
     # Runs in the root directory of this repository.
-    if [ "${USE_TX}" = true ]; then
+    if [ \( -n "$IS_OSX" -a "$MB_PYTHON_VERSION" = "2.7" \) -o \( "$MB_PYTHON_VERSION" = "3.6" \) ]; then
```

Comment:
ugly, but it should work. You didn't like the `env_vars.sh` thinghy?

Reply:
Not that I didn’t like it, I just still don’t understand how to get it to work. Anyway, the or branch didn’t work, so I’m just installing it for any 3.6 Python, on AppVeyor we check 2.7 anyway.

Reply:
This is absurd, even `MB_PYTHON_VERSION` is not getting passed to this function on Linux. I’m lost.

Reply:
It is annoying. I'll send a patch to upstream multibuild to fix this.

Reply:
OK, afdko installed but still tx is not found, grrr.